modules/level_set/examples/rotating_circle/area.py

```diff
@@ -0,0 +1,38 @@
+"""
+python /Users/slauae/projects/moose/modules/level_set/examples/rotating_circle/area.py
```

Comment:
Not a huge deal, but you should probably fix this comment to not refer to your home directory.